index.html

```diff
@@ -0,0 +1,160 @@
+    </head>
+    <Body style="background-color: #323232; color: azure; font-family: Arial, Helvetica, sans-serif; padding: 15x; padding-top: 30px">
+        <title>
+          Unoffical REV Port
+        </title>
+        <div id="wrapper">
+            <main>
+            <h1 align="center">
+                Unofficial REV Port
+            </h1>
+            <div class="center">
+                <p class="pg"> 
+                    Welcome to the website for the Unofficial REV Port team!
+                    <br><br>
+                    We are a team of FIRST students and alumni from both the FTC and FRC programs and 
+                    we have a passion for robotics and making. Our goal is to make FRC and FTC software 
+                    available to all, no matter their platform or operating system, as well as doing our best
+                    to keep current hardware relevant past the upcoming MRC switch for the 2027 FIRST season.
+                </p>
+            </div>
+            <div class="float-container">
+                <br>
+                <div class="float-child">
+                    <p style="padding: 3%;">
+                        <b>
+                            <a href="revhwc.html">REV Hardware Client</a>
+                        </b>
+                        <br><br>
+                        The REV Hardware Client is software for the configuration, updating, and management
+                        of REV Robotics Hardware. From REV, it only has a Windows x86_64 build, meaning it will not work for
+                        users with ARM CPUs or of Linux or macOS operating systems. 
+                        <br><br>
+                        Our port is currently in progress, and once
+                        finished we will have to work with REV on way to release once complete.
+                        <br>
+                        <div style="text-align: center;">
+                            <img src="revhardwareclient/rhwcmac.png" alt="a photo of the rev hardware client running on MacOS" style="width: 480px; height: 300px; margin: auto;">
+                        </div>
+                    </p>
+                </div>
+                
+                <div class="float-child">
+                    <p style="padding: 3%;">
+                        <b>
+                            <a href="revhubinterface.html">REV Hub Interface: Community Edition</a>
+                        </b>
+                        <br><br>
+                        The REV Hub Interface is software that can control a REV Expansion Hub over USB serial. REV's version
+                        has been useful for many teams, but has been mostly abandonded since at least 2022 and has several major bugs. It also only
+                        had a windwos x86_64 build. 
+                        <br><br>
+                        Our port is complete, featuring a new, more modern gui, several packaging option on Windows, MacOS, and Linux. 
```

Comment:
should be "Our port is complete. It features a new, modern GUI as well as several download options for Windows, Mac, and Linux."